packages/ticket_sidebar/src/App.tsx

```diff
@@ -70,11 +75,15 @@ const App = () => {
     fillTicketInfo(orderId)
   }, [orderId])
 
-  return orderId && orderId.length >= minimumOrderIdLength ? (
-    <OrderInfo orderId={orderId} />
-  ) : (
-    <p className="bold">Introduce un pedido para continuar</p>
-  )
+  if (isBetaTester) {
+    return orderId && orderId.length >= minimumOrderIdLength ? (
+      <OrderInfo orderId={orderId} />
+    ) : (
+      <p className="bold">Introduce un pedido para continuar</p>
+    )
+  } else {
+    return null
+  }
```

Comment:
Disclaimer: This comment is not meant to change current implementation! But to describe an interesting approach when using FFs.

I usually prefer reversing conditions when a FF is involved and use it in a _guard-fashion_. This way, when the FF is cleaned up, there will only be only removed lines. I.e:

```suggestion
  if (!isBetaTester) {
    return null
  }

  return orderId && orderId.length >= minimumOrderIdLength ? (
    <OrderInfo orderId={orderId} />
  ) : (
    <p className="bold">Introduce un pedido para continuar</p>
  )
```

Reply:
This is a much better approach that the one I took. I refactored the code.